src/osgPlugins/txp/trpage_managers.cpp

```diff
@@ -756,7 +758,6 @@ trpgPageManager::trpgPageManager()
     lastLoad = None;
     lastLod = 0;
     lastTile = 0;
-    scale = 1.0;
```

Comment:
lets leave this change in

Reply:
Readded that line.   Rest of file as-is.